css/styles.css

```diff
@@ -0,0 +1,368 @@
+	color: #fff;
+	background-color: #131d26;
+}
+
+body, button, input {
+	font-family: -apple-system, 'Inconsolata', Lato, sans-serif;
+	font-size: 1rem;
+}
+
+button, input {
+	outline: none;
+}
+
+button {
+	text-transform: uppercase;
+}
+
+*, ::after, ::before {
+   box-sizing: border-box;
+}
+
+.header {
+	position: fixed;
+	top: 0;
+	width: 100%;
+	z-index: 50;
+}
+
+.header:before {
+	content: '';
+	position: absolute;
+	top: 0;
+}
+
+.navbar {
+	display: flex;
+	padding: 10px 15px;
+	background-color: #070b0f;
+	line-height: 1;
+	align-items: center;
+	min-height: 56px;
+}
+
+.navbar a {
+	color: #fff;
+}
+
+.text_red {
+	color: #d9261c;
+}
+
+.navbar_brand {
+	font-size: 1.5rem;
+	font-family: 'Hammersmith One', arial;
+	text-decoration-line: none;
+	line-height: inherit; 
+	text-transform: uppercase;
+}
+
+.container {
+	position: relative;
+	display: block;
+	width: 100%;
+	padding: 10px 15px;
+	margin-left: auto;
+	margin-right: auto;
+	max-width: 1200px;
+}
+
+.card_list {
+	display: flex;
+	flex-direction: column;
+	margin-bottom: -15px;
+}
+
+.card {
+	display: flex;
+	padding: 15px 20px;
+	margin-bottom: 15px;
+	min-height: 100px;
+	border-radius: 10px;
+	background: linear-gradient(#3342a1, #003b79);
+	align-items: center;
+	justify-content: space-between;
+	flex-wrap: wrap;
+}
+
+.my_card {    
+	background: linear-gradient(#e21567, #d9261c);
+}
+
+.avatar {
+	width: 100%;
+	max-height: 65px;
+	max-width: 65px;
+	padding: 2px;
+	border-radius: 50%;
+	border: 3px solid rgba(255, 255, 255, 0.45);
+}
+
+@media (min-width: 768px) {
+	.avatar {
+		border-width: 5px;
+		max-height: 100px;
+		max-width: 100px;
+		padding: 3px;
+	}
+}
+
+.card__player_info {
+	margin-left: 10px;
+	margin-right: 10px;
+	line-height: 1.25rem;
+	font-size: 1.25rem;
+	flex-grow: 1;
+}
+
+.card__player_info .player_name {
+	font-weight: 500;
+}
+
+.card__player_info .player_money {
+	font-family: 'Hammersmith One', arial;
+	font-weight: 600;
+}
+
+.card .player_money:before {
+	display: inline-flex;
+	content: '$';
+	opacity: .5;
+	margin-top: 3px;
+	margin-left: -2px;
+	transform: scale(.75);
+}
+
+.card__player_info .player_money:after {
+	content: 'K';
+	opacity: .5;
+	font-size: 1rem;
+	margin-left: 2px;
+}
+
+.card__transfer_btn {
+	display: flex;
+	width: 48px;
+	height: 48px;
+	background: transparent;
+	border: 4px solid #7e92ef;
+	border-radius: 14px;
+	color: #fff;
+	font-size: 0;
+}
+
+.card__transfer_btn:hover {
+	background-color: rgba(255, 255, 255, 0.05);
+}
+.card__transfer_btn:focus {
+	background-color: rgba(255, 255, 255, 0.15);
+	box-shadow: 0 0 0 0.15rem rgba(126, 146, 239, 0.45);
+}
+
+.my_card .card__transfer_btn {
+	border-color: #ef7e7e!important;
+}
+
+.my_card .card__transfer_btn:focus {
+	box-shadow: 0 0 0 0.15rem rgba(239, 126, 126, 0.45);
+}
+
+@media (min-width: 768px) {
+	.card_list {
+		display: flex;
+		flex-direction: row;
+		justify-content: center;
+		flex-wrap: wrap;
+	}
+
+	.card {
+		margin-right: 7.5px;
```

Comment:
Non-integer pixel values are not really a good choice, because there is no such thing as a standard which prescribes browsers to treat this case specifically. This may result in different layout on different browsers.